requirements.txt

```diff
@@ -3,3 +3,4 @@ python-dateutil
 Flask
 PyYAML
 unicodecsv
+pymongo
```

Comment:
Can we move the pymongo dependency to `requirements-provider.txt` instead?

Reply:
I guess it could. :) Moved it.